feature/security/gnsi/certz/tests/client_certificates/client_certificates_test.go

```diff
@@ -0,0 +1,254 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client_certificates_test
+
+import (
+	context "context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+	"testing"
+
+	setupService "github.com/openconfig/featureprofiles/feature/security/gnsi/certz/tests/internal/setup_service"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+var (
+	testProfile = "gNxI"
+	serverAddr  string
+	username    = "certzuser"
+	password    = "certzpswd"
+)
+
+// createUser function to add an user in admin role.
```

Comment:
what tells this function that the user is an admin user?
are there cases where we'd want to add a 'not admin' user?
do we want 2 functions (or 3 or 4 or...) can't this function take as an argument the user role?

Reply:
@priyacj it doesn't look like you use this user to rotate certificates. I believe you use ondatra gnsi bindings for that purpose (which is the correct approach!)

So, the question is, do you really need a new user at all? Can't you use ondatra bindings for the verification as well?

Reply:
The test flow picks up the newly  created user after rotation of certificates to validate new connection  and not using the vendor specific credentials from the binding file.